RevenueCatUI/Templates/Components/PaywallComponentViewModel.swift

```diff
@@ -29,6 +28,7 @@ enum PaywallComponentViewModel {
 extension PaywallComponent {
 
     func toViewModel(
+        packageValidator: PackageValidator,
```

Comment:
This can get a default value of `PackageValidator()` so it _almost_ becomes an implementation detail of the `toViewModel()` function. This is a usual pattern I think with recursive functions, where the top level call gets a default argument, which it then passes down to every recursive call.

Reply:
@JayShortway I _did_ think about this but I almost don't want to because it really needs to accept that single high level instance of `PackageValidator`. I want to make sure that its not accidentally skipped when somebody creates a new view model 😬 

I'm kind of keeping it here for somebody adds another container component and needs to pass `packageValidator` into it and it causes a compile error if its missing 🤷‍♂️ Hopefully that makes sense 😅

Reply:
Yea works for me! I can always get behind a lil more explicitness.